src/features/game/expansion/LandExpansion.tsx

```diff
@@ -45,6 +45,7 @@ export const LandExpansion: React.FC = () => {
         <ScrollContainer
           className="bg-blue-300 overflow-scroll relative w-full h-full"
           innerRef={container}
+          ignoreElements={"*[data-prevent-drag-scroll]"}
```

Comment:
This allows us to set the data attribute `data-prevent-drag-scroll` on any element. This will prevent dragging the map scroll container when hovering on the element.